packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -38,6 +38,37 @@ type Props = {|
      * Makes a read-only input field that cannot be focused. Defaults to false.
      */
     disabled: boolean,
+
+    /**
+     * Provide a validation for the input value.
+     * Return a string error message or null | void for a valid input.
+     */
+    validation?: (value: string) => ?string,
```

Comment:
I know this is inherited from `TextField`, so I guess this is a suggestion for that component too. Since this is an action, perhaps it should be renamed to `validate` or `onValidate` and then change `onValidation` to `onValidated`. I think that would be clearer for folks using this API.

What do folks think?

Reply:
This makes senes, `"validate"` is more of an action word (aka a verb). Although, a lot of the TextFields built in Webapp use the name `"onValidation"` as well. I'm not sure if changing the name would complicate things when bringing WB `TextField`s to Webapp.

What do other people think?

Reply:
I think changing to `validate` makes sense. On the other hand, I think that changing to `onValidate` would make more sense instead of `onValidation`. In understand the meaning of changing to `onValidated`, but I think that `onValidate` adheres better to the way these event handlers are named in React (e.g. `onChange`, `onClick`, etc).

I think it's ok to rename all the occurrences in `webapp`, because we are going to also rename from `validator` to `validate` (or the prop name that we'll end up using).

Any thoughts?

Reply:
That seems fair enough. I will be changing `validation` --> `validate` and `onValidation` --> `onValidate`. I also updated a TextField ticket to add those changes later.